react/src/ config.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
This file seems to have a whitespace in the name `react/src/ config.json`

Reply:
ok